File: 2914-minimum-number-of-changes-to-make-binary-string-beautiful/2914-minimum-number-of-changes-to-make-binary-string-beautiful.cpp
class Solution {
public:
    int minChanges(string s) {
        int n=s.size();
        int ans=0;
        for(int i=0;i<n;i+=2){
           if(s[i]!=s[i+1]) ans++;
        }
        return ans;
    }
};

File: 2914-minimum-number-of-changes-to-make-binary-string-beautiful/2914-minimum-number-of-changes-to-make-binary-string-beautiful.js
/**
 * @param {string} s
 * @return {number}
 */
var minChanges = function(s) {
    let n = s.length;
    let i=0;
    let ans=0;
    while(i<n){
      if(s[i]!=s[i+1]) ans++;
      i+=2;
    }
    return ans;
};